### ISSUE TITLE
2 Speed Whole House Fan Controller Question

### ISSUE THREAD

**Issue by MoTechnicalities** [bwp91/homebridge-ewelink#467]
* **Please explain your feature request in a one or two sentences.**
@bwp91
I want the below-shown device to work as a HomeKit Fan controller for a 2-speed motor.
Would you be able to set up a configuration, as follows:
    **Channel 1** would be controlled by HomeKit Fan ON/OFF icon.
    **Channel 2** would be controlled by the HomeKit Fan's Speed control slider where Less than 51% is Channel 2 ON, and 51% to 100% is Channel 2 OFF.

![image](https://user-images.githubusercontent.com/68468472/218529209-c6605d7f-9b06-4b4d-96c6-a002f93b43ec.png)

For this devise:
U.S. Household 120v Hot Black Wire feeds power into COM1 Then only when HomeKit Fan accessory is ON, Channel-1 is ON, and power goes out NO1 wire into COM2.
Then, when HomeKit Fan speed slider is 1% to 50% Channel-2 is ON, sending power from COM2 out NO2 to the LOW speed motor wire, or

When HomeKit Fan speed slider is 51% to 100% Channel-2 is OFF,
sending power from COM2 out NC2 to the HIGH speed motor wire.

This setup would allow a householder to replace both the ON/OFF switch and the HIGH/OFF/LOW toggle found on most 2-speed Whole House Fans.

![image](https://user-images.githubusercontent.com/68468472/218656545-96595198-21d2-4a56-9cef-978e5d2703ce.jpeg)
![image](https://user-images.githubusercontent.com/68468472/218656640-f134455a-dac8-4e59-9667-8155c29fb4a0.jpeg)

* **Is your feature request related to a problem? Please describe.**
This is the best possible solution to control a 2-speed motor of this type that I could find.

* **Any particular eWeLink devices that this relates to?**
Newgoal 2-Channel Ewelink App Remote Control WiFi 433MHz RF Wireless Switch Relay

* **Anything else?**
I don't have the device yet but I want to know if you are able set up such a fan accessory configuration.

**Comment by bwp91** [comment]
hey, sorry for delay in replying, could be possible definitely, if still interested then please reopen ticket 🙂

**Comment by MoTechnicalities** [comment]
Hey Ben!
Thanks for responding.

After mulling it over in my head for a while, I decided it would be more desirable to do this with genuine HomeKit or Matter (when they become available) switches. The reason is because a whole-house fan is more of a permanent installation deal. It also needs physical switches. So I came up with this plan instead, as follows:

![image](https://user-images.githubusercontent.com/68468472/219827641-556a0751-46d1-4444-996a-aa20d4b2a2ab.jpeg)
